src/txkube/testing/integration.py

```diff
@@ -0,0 +1,25 @@
+# Copyright Least Authority Enterprises.
+# See LICENSE for details.
+
+"""
+Integration test generator for ``txkube.IKubernetesClient``.
+"""
+
+from zope.interface.verify import verifyObject
+
+from twisted.trial.unittest import TestCase
+
+from .. import IKubernetesClient
+
+
+def kubernetes_client_tests(get_kubernetes):
+    class KubernetesClientIntegrationTests(TestCase):
+        def test_interfaces(self):
+            """
+            The client provides ``txkube.IKubernetesClient``.
+            """
+            kubernetes = get_kubernetes(self)
+            client = kubernetes.client()
+            verifyObject(IKubernetesClient, client)
+
+    return KubernetesClientIntegrationTests
```

Comment:
It's fast and local-only right now but I anticipate adding real-Kubernetes-interacting tests here once IKubernetesClient has expanded to support any such things.